actions/add-watchtower.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+set -e
+
+cat > wtinput.json
```

Comment:
What are these three lines doing, creating a file, then removing it?

Reply:
I think it's needed so that jq can pull the URI from the action, then toss the file. 

I used the `import-umbel` action as a template.

Reply:
https://github.com/Start9Labs/lnd-wrapper/blob/master/actions/import-umbrel.sh